smart-common/src/main/java/org/smartdata/model/WhitelistHelper.java

```diff
@@ -0,0 +1,40 @@
+package org.smartdata.model;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.smartdata.conf.SmartConf;
+
+/**
+ * It's a helper for whitelist function. It's used in action and rule submit process.
+ * Especially, SmallFileScheduler also use this helper for  distinguishing between
+ * whitelist check and invalid small files exception.
+ */
+public class WhitelistHelper {
```

Comment:
I am thinking whether we can make this class a pure utility class, which means all methods are static methods and call them by this class name.